SRC/element/dispBeamColumn/DispBeamColumnNL3d.cpp

```diff
@@ -2343,10 +2343,9 @@ DispBeamColumnNL3d::getResistingForceSensitivity(int gradNumber)
 
       continue;
 
-      double si;
       for(int j = 0; j < order; j++) {
+	double si = s(j)*wt[i];
 	switch(code(j)) {
-	  si = s(j)*wt[i];
```

Comment:
This is the simplest possible fix, but it may be best to just remove everthing that follows `continue` to keep things clean.

Reply:
You can block comment the code, but don't remove.

Did you also fix the warning for `cmass` in the OPS function?

Reply:
I havent seen that warning, but I'm compiling from a branch that is missing some of the OPS functions.